### ISSUE TITLE
taggable/multiple allows duplicates in version 2.5.1

### ISSUE THREAD

**Issue by sainag** [sagalbot/vue-select#717]
Version 2.5.1 - Allowing duplicates. As shown below it doesn't highlight all the tags that have been selected. 
![vue-select 2 5 1 - multiple taggable - doesn t remember all](https://user-images.githubusercontent.com/1840739/50322839-1674d980-052b-11e9-8d31-90f24fad6e65.PNG)

However, version 2.4.0 doesn't allow duplicates. As shown below it does highlight all the tags selected and allows me to delete if I select or type in the same tag.
![vue-select 2 4 0 - multiple taggable - remembers all selected ones](https://user-images.githubusercontent.com/1840739/50322850-1e347e00-052b-11e9-87fa-ab39ca10ccf5.PNG)

Any suggestions to fix this in 2.5.1 because I need the selectontab option from 2.5.1 so I want to continue using 2.5.1?

**Comment by zamronypj** [comment]
I also experience same issue when using ajax. It allows duplicate items to be selected. My workaround right now is to downgrade to 2.4.0

**Comment by Christilut** [comment]
Same problem here if you use options with a `label` and `value`.

**Comment by josephzidell** [comment]
Bump

**Comment by kant312** [comment]
Bump

**Comment by TheJaredWilcurt** [comment]
Bump

* [Live Example](https://codepen.io/TheJaredWilcurt/pen/ErzGvE)

**Comment by wjonker85** [comment]
Hey All. I have also this problem. I did not know there was already an item opened for this. I have made an example showing the problem. The example can be seen on https://codepen.io/anon/pen/oVWMKK.

I will try downgrading to v 2.4.0. Thanks for the advice.

**Comment by wjonker85** [comment]
Downgrading to version 2.4.0 did resolve the problem. Version 2.4.1 is also affected by this problem.

**Comment by sagalbot** [comment]
This is a regression introduced in 2.5.1. I'm drafting a release for 2.6 this morning that will address this.

**Comment by wjonker85** [comment]
> This is a regression introduced in 2.5.1. I'm drafting a release for 2.6 this morning that will address this.

Thank you so much for resolving this issue :).

**Comment by sagalbot** [comment]
Fixed in 2.6 – this took way longer than it should have!

**Comment by sainag** [comment]
Thanks for resolving @sagalbot
